crates/typst/src/symbols/sym.rs

```diff
@@ -341,18 +341,18 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     // Set theory.
     emptyset: [
         '∅',
+        arrow.r: '⦳',
+        arror.l: '⦴',
```

Comment:
Sorry for the late review, but I think there is a typo here (and below as well), isn't it supposed to be `arrow` and not `arror`?

Reply:
You're right. This is annoying. I'll make another PR to fix that, unless this can be included in something else.

Reply:
Fixed by #5055.